Span stack trace min duration (#1498)
elastic/apm-agent-python: docs/configuration.asciidoc

To disable the limit and always capture all frames, set the value to `-1`.

[float]
[[config-span-stack-trace-min-duration]]
==== `span_stack_trace_min_duration`

<<dynamic-configuration, image:./images/dynamic-config.svg[] >>

[options="header"]
|============
| Environment                                 | Django/Flask                    | Default
| `ELASTIC_APM_SPAN_STACK_TRACE_MIN_DURATION` | `SPAN_STACK_TRACE_MIN_DURATION` | `"5ms"`
|============

> By default, the APM agent will collect a stack trace with every recorded span
> which has a duration equal to or longer than this configured threshold.  While

Reviewer: This sentence should use "that" instead of "which". The phrase, "_has a duration equal to or longer than this configured threshold_" is an essential clause, i.e. one that must be present for the sentence to be accurate. "which" should only be used to include information that is non-essential to the meaning of a sentence. 

```suggestion
By default, the APM agent collects a stack trace with every recorded span
that has a duration equal to or longer than this configured threshold.  While
```

stack traces are very helpful to find the exact place in your code from which a
span originates, collecting this stack trace does have some overhead. Tune this

> threshold to ensure that you are only collecting stack traces for spans which

Reviewer: ```suggestion
threshold to ensure that you only collect stack traces for spans that
```

could be problematic.

> To collect traces for all spans, independent of the length, set the value to `0`.

Reviewer: What about something like this instead:

```suggestion
To collect traces for all spans, regardless of their length, set the value to `0`.
```

| `ELASTIC_APM_SPAN_FRAMES_MIN_DURATION` | `SPAN_FRAMES_MIN_DURATION` | `"5ms"`
|============

> NOTE: this config value is being deprecated. Use

Reviewer: ```suggestion
NOTE: This config value is being deprecated. Use
```
